Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.13-alpine as builder
+ARG CLI_VERSION=v0.0.1
+WORKDIR /app
+ADD . /app
```

Comment:
action - don't add everything in the image, only what you need. It will help using the docker cache. ;)